apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.html.ts

```diff
@@ -46,91 +36,53 @@ export function InstructorAssessmentAccess({
-                                >
-                                  Hidden
-                                </a>
-                              `}
-                        </td>
-                        <td>${access_rule.start_date}</td>
-                        <td>${access_rule.end_date}</td>
-                        <td>${access_rule.active}</td>
-                        <td>${access_rule.credit}</td>
-                        <td>${access_rule.time_limit}</td>
-                        <td>${access_rule.password}</td>
-                        <td>
-                          ${access_rule.pt_exam_name
-                            ? html`
-                                <a
-                                  href="${resLocals.config
-                                    .ptHost}/pt/course/${access_rule.pt_course_id}/staff/exam/${access_rule.pt_exam_id}"
-                                >
-                                  ${access_rule.pt_course_name}: ${access_rule.pt_exam_name}
-                                </a>
-                              `
-                            : access_rule.exam_uuid
-                              ? resLocals.devMode
-                                ? access_rule.exam_uuid
-                                : html`
-                                    <span class="text-danger">
-                                      Exam not found: ${access_rule.exam_uuid}
-                                    </span>
-                                  `
-                              : html`&mdash;`}
-                        </td>
-                      </tr>
-                    `;
-                  })}
-                </tbody>
-              </table>
+            <div
+              class="card-header bg-primary text-white d-flex align-items-center justify-content-between"
+            >
+              <div class="col-auto">
+                ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Access
+              </div>
+              ${resLocals.authz_data.has_course_instance_permission_edit
+                ? html`
+                    <div class="col-auto">
+                      <button id="enableEditButton" class="btn btn-sm btn-light">
+                        <i class="fa fa-edit" aria-hidden="true"></i> Edit Access Rules
+                      </button>
+                      <span id="editModeButtons" style="display: none">
+                        <button id="saveAndSyncButton" class="btn btn-sm btn-light" type="button">
+                          <i class="fa fa-save" aria-hidden="true"></i> Save and sync
+                        </button>
+                        <button class="btn btn-sm btn-light" onclick="window.location.reload()">
+                          Cancel
+                        </button>
+                        <p class="mb-0"></p
```

Comment:
What's this?

Reply:
Not sure what happened here. Removed in [1dfef07](https://github.com/PrairieLearn/PrairieLearn/pull/9934/commits/1dfef07527ed0296ff3cdfa7c58d85fce2304b97).